client/js/components/home.jsx

```diff
@@ -149,75 +179,98 @@ class Home extends React.Component {
     if(!_.isEmpty(assessOffered)){
       let qBankHost = this.props.settings.qBankHost ? this.props.settings.qBankHost : "https://qbank-clix-dev.mit.edu";
       let playerHost = this.props.settings.assessmentPlayerUrl; //This will need to be the instance deployed, not localhost.
-      let url = encodeURI(`${playerHost}/?unlock_next=ON_CORRECT&api_url=${qBankHost}/api/v1&bank=${assessOffered.bankId}&assessment_offered_id=${assessOffered.id}#/assessment`);
+      let url = `${playerHost}/?unlock_next=ON_CORRECT&api_url=${qBankHost}/api/v1&bank=${assessOffered.bankId}&assessment_offered_id=${assessOffered.id}#/assessment`;
       return `<iframe src="${url}"/>`;
     } else {
       return "";
     }
   }
 
-  breadcrumbs(hierarchy){
-    var checked = _.compact(_.map(this.state.itemChecked, (val, key)=>{if(val === true){return key;}}));
-    return _.map(hierarchy, (bank)=>{
-      if(_.includes(checked, bank.id)){
-        return [(
-          <div className="c-breadcrumb" key={bank.id}>
-            <span>{bank.displayName.text}</span>
-            <a href="#" onClick={()=>{this.checkItem(bank, false);}}>
-              <svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48">
-                  <path d="M29.17 16l-5.17 5.17-5.17-5.17-2.83 2.83 5.17 5.17-5.17 5.17 2.83 2.83 5.17-5.17 5.17 5.17 2.83-2.83-5.17-5.17 5.17-5.17-2.83-2.83zm-5.17-12c-11.05 0-20 8.95-20 20s8.95 20 20 20 20-8.95 20-20-8.95-20-20-20zm0 36c-8.82 0-16-7.18-16-16s7.18-16 16-16 16 7.18 16 16-7.18 16-16 16z"/>
-              </svg>
-            </a>
-          </div>
-        ), this.breadcrumbs(bank.childNodes)];
-      }
-    });
+  slidingClasses(){
+    if(this.state.isOpen){
+      return "o-admin-container  o-admin-container--preview is-open";
+    }
+    return "o-admin-container  o-admin-container--preview";
   }
 
-  spinner(){
+  iframeCode(){
+    if(this.state.openIframe){
+      return (
+      <div className="c-preview-embed">
+        <label for="embed">Embed Code</label>
+        <textarea id="embed" value={this.iframe()} readOnly="true" ></textarea>
```

Comment:
It is a string in the Embed code area. So that they can copy and paste the "iframe code" and put it in the epub.